composer.json

```diff
@@ -12,7 +12,7 @@
     "xp-framework/tokenize": "^8.0",
     "xp-forge/json": "^3.1",
     "xp-forge/uri": "^1.3",
-    "xp-forge/marshalling": "^0.2",
+    "xp-forge/marshalling": "^0.3",
```

Comment:
To prevent conflicts when running "composer up", I'd suggest using `^0.3 | ^0.2`. Yes, and we should probably move xp-forge/Marshalling to a 1.0-release soon:)